src/js/review/ArrayField.jsx

```diff
@@ -211,7 +211,7 @@ class ArrayField extends React.Component {
                     <div className="small-12 columns va-growable-expanded">
                       {isLast
                         // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
-                        ? <h5 tabIndex="0" ref={this.handleNewItemTitleMounted}>New {uiOptions.itemName || 'Item'}</h5>
+                        ? <h5 className="schemaform-array-row-title" tabIndex="0" ref={this.handleNewItemTitleMounted}>New {uiOptions.itemName || 'Item'}</h5>
```

Comment:
@jbalboni I thought schemaform fits in with what's already here but we only style stuff called `form-` so maybe this hsould be `form-review-array-row-title` or `-item-title`?

Reply:
I think you can still use `schemaform`; we do style classes with that prefix, the styles for that are just in vets-website.

Reply:
Thanks I've merged the style change into formation. This should be ready to go.

Reply:
We should talk about how to migrate this codebase back into the VA site. The classes being applied here should technically coordinate with the CSS that the USFS library ships and I don't think that's happening lately. Things are getting further out of sync which will get confusing for anyone who wants to pick up USFS and change it later.

Reply:
I think once we move the us-forms-system back into `vets-website` we can delete the `1.x-stable` branch